src/app/app.component.scss

```diff
@@ -15,10 +15,28 @@
 
 .presentation-wrapper {
     float: left;
-    width: 65%;
+    width: 50%;
+    
 }
 
 .upcoming-events-wrapper {
     float: right;
-    width: 32%;
+    width: 50%;
+}
+
+ p {
+    text-align: center;
+    font-size: 25px;
+    margin-bottom: 50px;
+    font-family: 'Fredoka One', cursive;

```

Comment:
Personal Opinion: Fredoka One seems to be a little dark/heavy especially on smaller devices.
We should limit the number of fonts to 2 maximum.    I am open to an overall font change but let's keep in mind all screen sizes when choosing.

Reply:
I agree with that!  I was just trying to find something a bit different so that we could start a conversation about fonts.

Reply:
Cool, your PR has started that conversation among the core organizers.   Mission accomplished.
We are reaching out to some designers to see if anyone would like to provide a formal full page design to solidify our desires allowing everyone to contribute better to the overall goal.